tests/Fixer/ConstantNotation/NativeConstantInvocationFixerTest.php

```diff
@@ -189,6 +189,24 @@ public function provideFixWithDefaultConfigurationCases()
 #
 M_PI;
 echo M_PI;
+',
+            ],
+            [
+                '<?php foo(\E_DEPRECATED | \E_USER_DEPRECATED);',
+                '<?php foo(E_DEPRECATED | E_USER_DEPRECATED);',
+            ],
+            [
+                '<?php
+try {
+    foo(\JSON_ERROR_DEPTH|\JSON_PRETTY_PRINT|\JOB_QUEUE_PRIORITY_HIGH);
+} catch (\Exception | \InvalidArgumentException|\UnexpectedValueException|LogicException $e) {
+}
+',
+                '<?php
+try {
```

Comment:
@ntzm your work looks good :)

However the tests are failing, this is because these tests should only run on PHP7.1 or up, below this syntax is not supported. Please update than we are close to merging :+1:

Reply:
Oops, my bad